tests/server/routes.js

```diff
@@ -35,13 +35,14 @@ define([
     '/signup_complete': { statusCode: 200 },
     '/confirm': { statusCode: 200 },
     '/settings': { statusCode: 200 },
-    '/settings/avatar': { statusCode: 200 },
     '/settings/avatar/change': { statusCode: 200 },
     '/settings/avatar/gravatar': { statusCode: 200 },
     '/settings/avatar/camera': { statusCode: 200 },
     '/settings/avatar/crop': { statusCode: 200 },
     '/settings/avatar/gravatar_permissions': { statusCode: 200 },
-    '/change_password': { statusCode: 200 },
+    '/settings/change_password': { statusCode: 200 },
+    '/settings/delete_account': { statusCode: 200 },
+    '/settings/display_name': { statusCode: 200 },
```

Comment:
Can you add `/settings/communication_preferences` for completeness, at least if we are using `en-US` for tests.